### DISCUSSION TITLE
DB2 for IBMi

### DISCUSSION THREAD

**Discussion by eshaw21** [codefori/vscode-ibmi#1545] [General]
Hi all! 

I have been using Code for IBMi for awhile and decided to try out DB2 for IBMi. I gave it permissions needed to create the IFS directory etc. 

When I connect via Code for IBMi, I click on DB2 for IBMi, it starts the SQL job but I never see my database. 

See screenshot below of what I am seeing and please point me in any direction! 

![image](https://github.com/halcyon-tech/vscode-ibmi/assets/131499740/8dd8f156-e11e-4ddf-ac76-f2a4fac2bb4c)

**Comment by worksofliam** [comment]
Hi! From the Schema Browser you can use this button (which you will see when you hover your mouse in the Schema Browser window) which will load a list of available schemas: 

<img width="322" alt="image" src="https://github.com/halcyon-tech/vscode-ibmi/assets/3708366/167d58a1-3883-4073-8bc3-404659215547">

**Reply by eshaw21** [reply]
@worksofliam  thank you so much, I knew it was something simple but I was lost.
